pkg/services/correlations/models.go

```diff
@@ -42,21 +49,31 @@ type CorrelationConfig struct {
 	// required:true
 	// example: { "expr": "job=app" }
 	Target map[string]interface{} `json:"target" binding:"Required"`
+	// Source data transformations
+	// required:false
+	// example: [{"type": "logfmt"}]
+	Transformations []Transformation `json:"transformations,omitempty"`
```

Comment:
Could you also please update backend tests? (model validation in `models_test.go` and CRUD updates `correlations_(create/update/delete/read)_test.go`?